docs/index.html

```diff
@@ -10,72 +10,68 @@
     <title>CtTableFor: Rails Responsive Table. table_for helper</title>
   </head>
 <body>
+<div class="container">
```

Comment:
As a general comment, for a doc, is more important to be understandable than any other thing. Try not to use periphrastic sentences and be redundant. Keep it as simple as you can:
- To make this, add this
- To show that, add this attributes, like this example.

If you need help to translate or explain some feature, ping me.